SharpBucket/V2/Pocos/Issue.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace SharpBucket.V2.Pocos
+{
+    public class Issue
+    {
+        public int Id { get; set; }
```

Comment:
I known it's a bit ugly but can you write the properties in camelCase in this class
Otherwise when you will implement the POST operation it won't works

The json serialization is not really well done in that project, we should do something at some point.
Don't know if you can help later on that but for now let just try to have this endpoint works.

Reply:
Done.